pkg/builder/builder.go

```diff
@@ -478,6 +479,16 @@ func buildScrutinizeVolumes(vscr *v1beta1.VerticaScrutinize, vdb *vapi.VerticaDB
 		secrets.IsK8sSecret(vdb.Spec.NMATLSSecret) {
 		vols = append(vols, buildNMACertsSecretVolume(vdb))
 	}
+	// we add a volume for the password when the password secret
+	// is on k8s
+	if vdb.Spec.PasswordSecret != "" &&
+		secrets.IsK8sSecret(vdb.Spec.PasswordSecret) {
+		projSources := []corev1.VolumeProjection{
+			// If these is a superuser password, include that in the projection
+			{Secret: buildPasswordProjection(vdb)},
+		}
+		vols = append(vols, buildVolumeFromProjections(projSources, scrutinizeDBpasswordMountName))
```

Comment:
Does the password need to be stored in a project volume? It would be simple if its just a regular volume.